test/functional/api/v2/users_controller_test.rb

```diff
@@ -1,13 +1,15 @@
 require 'test_helper'
 
 class Api::V2::UsersControllerTest < ActionController::TestCase
-  valid_attrs = { :login => "johnsmith",
+  def setup
+    setup_users
+  end
+
+  let(:valid_attrs) do
+    { :login => "johnsmith",
```

Comment:
nah, I need to remove this entirely... it got in by accident, something funky on my machine...